.github/workflows/check-documentation-urls.yml

```
@@ -26,7 +26,7 @@ jobs:
         run: pnpm install --frozen-lockfile
 
       - name: Build nodes-base
```

The name `Build nodes-base` doesn't really match the functionality anymore. Could we rename it to something that reflects the reality better?

```suggestion
      - name: Build depended packages
```

or similar